File: app/src/main/java/com/example/a18/path/transition/TransitionActivity.java
package com.example.a18.path.transition;

import android.support.v4.app.Fragment;

import com.example.a18.path.TabViewPagerActivity;

public class TransitionActivity extends TabViewPagerActivity {


    @Override
    public Fragment[] getFragments() {
        return new Fragment[]{
            new ConstrainSetFragment(),
            new LifelineFragment(),
            new AutoFragment(),
            new ChangeBoundsFragment(),
            new PathFragment(),
            new SlideSample(),
            new ScaleSample(),
        };
    }
}

File: m_transition/src/main/java/pdog18/com/module_transition/TransitionActivity.kt
package pdog18.com.module_transition

import pdog18.com.base.BaseFragment
import pdog18.com.base.TabViewPagerActivity

class TransitionActivity : TabViewPagerActivity() {

    override fun getFragments(): Array<BaseFragment> {
        return arrayOf(ConstrainSetFragment(), LifelineFragment(), AutoFragment(), ChangeBoundsFragment(), PathFragment(), SlideSample(), ScaleSample())
    }
}
